### ISSUE TITLE
Mud ball recipe clash

### ISSUE THREAD

**Issue by gruff8** [EnigmaticaModpacks/Enigmatica6#3824]
### Modpack Version

0.5.21 - expert mode

### Describe your issue.

Mud ball from environmental has a recipe clash with mud block from "Oh the biomes you'll go".
Mud bricks cannot be made from mud balls from "Oh the biomes you'll go" (which are made from the mud blocks in the recipe clash).

Fix would be to block one of the recipes or ore dictionary the mud balls from each mod.

Or am I missing something in how I am supposed to get the mud brick?

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by MuteTiefling** [comment]
https://github.com/NillerMedDild/Enigmatica6/pull/3823

**Comment by MuteTiefling** [comment]
Fix inbound

**Comment by MuteTiefling** [comment]
I realize this touches the normal release version, however If you're testing Expert, I'd really recommend pulling straight from github here to get all the latest changes as we continue to develop it: https://github.com/NillerMedDild/Enigmatica6#testing

**Comment by DoctorScorch** [comment]
Was this already implemented in dev? I just updated to apply the latest changes on the dev branch and still couldn't make mud bricks.

What I tried to do was craft the mudballs from environmental which instead crafts the mud blocks from Oh The Biomes You'll Go due to a recipe conflict, which is fine since we can turn them into mudballs, except the recipe still outputs mudballs from OTBYG and they were not smeltable to make the bricks. Also tried to use the Stonecutter to change any of the mud blocks into other ones, unfortunately it didn't work.

Just reporting the process in case it is helpful or if someone can tell if I'm doing something wrong
